src/ProfStef-Core/StPlaygroundPreviousProfStefCommand.class.st

```diff
@@ -0,0 +1,30 @@
+Class {
+	#name : 'StPlaygroundPreviousProfStefCommand',
+	#superclass : 'StPlaygroundCommand',
+	#category : 'ProfStef-Core-Command',
+	#package : 'ProfStef-Core',
+	#tag : 'Command'
+}
+
+{ #category : 'default' }
+StPlaygroundPreviousProfStefCommand class >> defaultDescription [
+
+	^ 'go to the previous lessons'
```

Comment:
Do we go to the previous one (no s needed) or do we go to multiple previous lessons?